File: Assetraz/AssetrazAPI/AssetrazDataProvider/Entities/ScrapList.cs
using System;
using System.Collections.Generic;

namespace AssetrazDataProvider.Entities
{
    public partial class ScrapList
    {
        public Guid ScrapAssetId { get; set; }
        public Guid InventoryId { get; set; }
        public Guid? RefurbishedAssetId { get; set; }
        public DateTime RefurbishedDate { get; set; }
        public string RefurbishedBy { get; set; } = null!;
        public string Status { get; set; } = null!;
        public bool? RemovedFromScrap { get; set; }
        public DateTime? ScrappedDate { get; set; }
        public string? Remarks { get; set; }
        public DateTime CreatedDate { get; set; }
        public string CreatedBy { get; set; } = null!;
        public DateTime? UpdatedDate { get; set; }
        public string? UpdatedBy { get; set; }

        public virtual Inventory Inventory { get; set; } = null!;
    }
}

File: Assetraz/AssetrazUI/src/types/ScrapList.ts
export type ScrapList = {
    scrapAssetId: string;
    categoryName: string;
    manufacturerName: string;
    modelNumber: string;
    warrentyDate: Date;
    serialNumber: string;
    assetTagNumber: string;
    scrappedDate: Date;
    markedBy: string;
    status: string;
    remarks: string;
    networkCompanyName: string;
    inventoryId: string;
    refurbishedAssetId?: string;
};
